stdlib/xml/dom/minidom.pyi

```diff
@@ -30,13 +30,33 @@ class Node(xml.dom.Node):
     def localName(self) -> str | None: ...
     def __bool__(self) -> Literal[True]: ...
     if sys.version_info >= (3, 9):
-        def toxml(self, encoding: str | None = None, standalone: bool | None = None) -> str: ...
+        @overload
+        def toxml(self, encoding: str, standalone: bool | None = None) -> bytes: ...
+        @overload
+        def toxml(self, encoding: None = None, standalone: bool | None = None) -> str: ...
+        def toxml(self, encoding: str | None = None, standalone: bool | None = None) -> str | bytes: ...
+        @overload
+        def toprettyxml(
+            self, indent: str = "\t", newl: str = "\n", encoding: str = ..., standalone: bool | None = None
+        ) -> bytes: ...
+        @overload
+        def toprettyxml(
+            self, indent: str = "\t", newl: str = "\n", encoding: None = None, standalone: bool | None = None
+        ) -> str: ...
         def toprettyxml(
             self, indent: str = "\t", newl: str = "\n", encoding: str | None = None, standalone: bool | None = None
         ) -> str: ...
```

Comment:
```suggestion
        ) -> str | bytes: ...
```

Reply:
In fact this should be deleted; stubs don't need an implementation, only the overloads.